src/config.rs

```diff
@@ -46,6 +46,8 @@ pub struct Config {
     /// List of programs to ignore.
     #[serde(default)]
     pub program_ignores: Vec<String>,
+    #[serde(default)]
+    pub owners: Vec<String>,
```

Comment:
For the sake of consistency, can we call this `program_filters`? We can document in the usage section that this is the same as the account owner for clarity.